IFComp/root/src/ballot/vote.tt

```diff
@@ -22,7 +22,7 @@
 [% IF c.user %]
     [% IF c.user.get_object.current_comp_entries %]
```

Comment:
It looks like this will only display the message to main authors, not co-authors. Should it be displayed to both?

Likewise, there is a similar message in `ballot/index.tt` that needs to be updated, and probably displayed to both authors and co-authors.